notes/stv/20220216-test-deploy-01.txt

```diff
@@ -0,0 +1,225 @@
+changed: [localhost] => {"changed": true, "cmd": "python3 /tmp/run-test.py | tee /tmp/test-result.json", "delta": "2:38:47.353090", "end": "2022-02-16 15:40:20.254404", "rc": 0, "start": "2022-02-16 13:01:32.901314", "stderr": "", "stderr_lines": [], "stdout": "Test completed after: 9527.15 seconds\n{'SetUp': {'totaltime': '40.78', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'Mean_proper_motions_over_the_sky': {'totaltime': '56.25', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'Source_counts_over_the_sky.json': {'totaltime': '20.76', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'Good_astrometric_solutions_via_ML_Random_Forrest_classifier': {'totaltime': '553.25', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'QC_cuts_dev.json': {'totaltime': '4493.28', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'WD_detection_dev.json': {'totaltime': '4356.36', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'Library_Validation.json': {'totaltime': '6.45', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}}", "stdout_lines": ["Test completed after: 9527.15 seconds", "{'SetUp': {'totaltime': '40.78', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'Mean_proper_motions_over_the_sky': {'totaltime': '56.25', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'Source_counts_over_the_sky.json': {'totaltime': '20.76', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'Good_astrometric_solutions_via_ML_Random_Forrest_classifier': {'totaltime': '553.25', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'QC_cuts_dev.json': {'totaltime': '4493.28', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}, 'WD_detection_dev.json': {'totaltime': '4356.36', 'status': 'SLOW', 'msg': '', 'valid': 'TRUE'}, 'Library_Validation.json': {'totaltime': '6.45', 'status': 'SUCCESS', 'msg': '', 'valid': 'TRUE'}}"]}
+
+
+
+# Results [Formatted]
+
+{
+	'SetUp': {
+		'totaltime': '40.78',
+		'status': 'SUCCESS',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'Mean_proper_motions_over_the_sky': {
+		'totaltime': '56.25',
+		'status': 'SLOW',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'Source_counts_over_the_sky.json': {
+		'totaltime': '20.76',
+		'status': 'SUCCESS',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'Good_astrometric_solutions_via_ML_Random_Forrest_classifier': {
+		'totaltime': '553.25',
+		'status': 'SLOW',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'QC_cuts_dev.json': {
+		'totaltime': '4493.28',
+		'status': 'SUCCESS',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'WD_detection_dev.json': {
+		'totaltime': '4356.36',
+		'status': 'SLOW',
+		'msg': '',
+		'valid': 'TRUE'
+	},
+	'Library_Validation.json': {
+		'totaltime': '6.45',
+		'status': 'SUCCESS',
+		'msg': '',
+		'valid': 'TRUE'
+	}
+}
+
+
+# Note: WD Detections allways seems to be slower than our benchmarks expect. Do we need to update the value for this?
```

Comment:
We need to review what these benchmark values mean and how to manage changes to them.
If we just change a value now, then what does that mean for all the tests run before this point ?